srcpkgs/ibus-libpinyin/template

```diff
@@ -0,0 +1,18 @@
+# Template file for 'ibus-libpinyin'
+pkgname=ibus-libpinyin
+version=1.10.92
+revision=1
+build_style=gnu-configure
+configure_args="--prefix=/usr"
```

Comment:
--prefix=/usr is automatically passed by the `gnu-configure` build style

Reply:
Awesome. Will be removed in next commit also.